### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by vbkaisetsu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,13 @@ Dependency
 
 * C++ primitiv
 * Python 3 (3.5 or later)
+* NumPy
```
Comment (odashi) [inline README.md]: Which version is supported?

File: README.md [hunk 2]
```diff
@@ -6,12 +6,13 @@ Dependency
...
 * Cython (0.27 or later)
 
 How to install?
 ---------------------------------
 
-1. Install C++ primitiv:
+1. Install [C++ primitiv](http://github.com/primitiv/primitiv):
```
Comment (odashi) [inline README.md]: "C++ primitiv" -> "primitiv core library"

File: README.md [hunk 3]
```diff
@@ -31,12 +32,12 @@ Currently, Cython 0.27 is not contained in Debian and Ubuntu repositories.
...
 $ python3 ./setup.py test  # (optional)
-$ sudo python3 ./setup.py install
+$ sudo python3 ./setup.py install [--enable-cuda] [--enable-opencl]
 ```
 
 You also can use `LIBRARY_PATH` and `CPLUS_INCLUDE_PATH` depending on your environment.
 
-CUDA is not available in the default settings. Please add `--enable-cuda` option
-to all commands above if you want to enable CUDA.
+CUDA and OpenCL are not available in the default settings. Please add `--enable-cuda` and/or
```
Comment (odashi) [inline README.md]: It might be better to remove negation sentences (namely sentences including "not") from documents.

File: README.md (hunk 1 above)
Comment (vbkaisetsu) [reply README.md]: I couldn't find the actual minimum required version, so I added the version that contained in Ubuntu 16.04.

**Comment by vbkaisetsu** [comment]
Fixed. Thanks.

**Pull request merged**